source/args.c

```diff
@@ -247,7 +247,7 @@ int parse_args(struct args *args, int argc, const char *argv[])
 		case OPTION_TARGET:
 			if ((parse_addr(&args->target, optarg)) < 0)
 				return -1;
-			
+
```

Comment:
This is a formatting change. If there are many of those, it makes more sense to have this as a separate commit: "formatting: remove whitespace from empty lines".